lang/en/texts/mobile-app-open-food-facts.html

```diff
@@ -57,4 +59,29 @@ <h1>Install the Open Food Facts mobile app</h1>
 </scripts>
 
 <initjs>
-</initjs>
\ No newline at end of file
+</initjs>
+
+<br />
+
+<div class="panel callout">
+<h2>It's new !</h2>
+
+<p>
+<b>July 2019 :</b> <a href="https://www.helloasso.com/associations/open-food-facts/collectes/une-meilleure-alimentation-pour-une-meilleure-sante-on-se-mobilise">A better diet for better health: let's mobilize!</a><br>Crowdfunding campaign to imagine and create together the ideal app!</p>
+<a href="https://www.helloasso.com/associations/open-food-facts/collectes/une-meilleure-alimentation-pour-une-meilleure-sante-on-se-mobilise" class="button">Support the development of the app</a> <br />
```

Comment:
```suggestion
<a href="https://www.helloasso.com/associations/open-food-facts/collectes/une-meilleure-alimentation-pour-une-meilleure-sante-on-se-mobilise" class="button" hreflang="fr">Support the development of the app (French)</a><br>
```
See above. Also, couldn't we just link to [Open Collective](https://opencollective.com/openfoodfacts-server)?